src/global.css

```diff
@@ -67,3 +67,37 @@ button:not(:disabled):active {
 button:focus {
   border-color: #666;
 }
+
+.popup {
+  min-width: 180px;
+  padding-right: 12px;
+  max-height: 240px;
+  overflow: scroll;
```

Comment:
Could we change this to

```suggestion
  overflow: auto;
```

At least on my (ubuntu) machine I get scrollbars no matter what:

![image](https://user-images.githubusercontent.com/375002/169309400-8bb4e968-07fe-4d7d-b33f-79ea80c79f2e.png)